File: LEETCODE/面试题58 - II. 左旋转字符串.java
class Solution {
    public String reverseLeftWords(String s, int n) {
        int len = s.length();
        return s.substring(n, len) + s.substring(0,n);
    }
}

class Solution {
    public String reverseLeftWords(String s, int n) {
        String res = "";
        for(int i=n;i<s.length();i++) {
            res += s.charAt(i);
        }
        for(int i=0;i<n;i++) {
            res += s.charAt(i);
        }
        return res;
    }
}

File: LEETCODE/kotlin/面试题58 - II. 左旋转字符串.kt
class Solution {
    fun reverseLeftWords(s: String, n: Int): String {
        var len = s.length
        return s.substring(n, len) + s.substring(0,n)
    }
}

class Solution {
    fun reverseLeftWords(s: String, n: Int): String {
        var res = ""
        for(i in n..s.length-1) {
            res += s[i]
        }
        for(i in 0..n-1) {
            res += s[i]
        }
        return res
    }
}